config/locales/en.yml

```diff
@@ -23,7 +23,7 @@ en:
  navigation:
    about: 'About'
    blog: 'Blog'
-   contributors: 'Contributors'
+   contributors: 'Contribute'
```

Comment:
If we're going to take this "Contribute" route, let's also rename the key to "contribute"

Reply:
We'll also need to translate this in all of our locales as well.

Reply:
I will edit the key in all locales but we need to open another issue for translation.